src/pages/Home/Home.jsx

```diff
@@ -1,5 +1,10 @@
 import "../Home/Home.css";
-import { Sidebar, VideoCard } from "../../components";
+import { Sidebar, CategoryCard } from "../../components";
+import Ballroom from "../../assets/ballroom.jpg";
+import Contemprary from "../../assets/contemprary.jpg";
+import HipHop from "../../assets/hip-hop.jpg";
+import Banner from "../../assets/banner.jpg";
```

Comment:
you can make an index.js file in the assets folder to store all images and then you can easily import them with one line of code anywhere .